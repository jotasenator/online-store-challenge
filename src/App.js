
import CreateItem from './screen/CreateItem'
import Table from './screen/Table'

function App() {
  return (
    <div className="App">
      <Table />
      <CreateItem />

    </div>
  );
}

export default App;
