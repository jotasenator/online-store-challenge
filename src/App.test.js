import React from 'react'
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import App from './App'

describe('Snapshot from <App/>', () => {

    test('should render <App/> correctly', () => {
        const wrapper = shallow(<App />)
        expect(wrapper).toMatchSnapshot()
    })

})

