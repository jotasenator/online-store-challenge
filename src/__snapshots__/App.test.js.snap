// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot from <App/> should render <App/> correctly 1`] = `
<div
  className="App"
>
  <Routes />
</div>
`;
